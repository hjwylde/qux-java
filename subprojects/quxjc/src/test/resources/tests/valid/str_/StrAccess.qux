package valid.str_

str f(str foo):
    foo = 'qux'

    return foo

void main([str] args):
    foo = 'bar'
    print foo[0]
    print foo[1]
    print foo[2]
    print f(foo)[0]
    print foo[0]

