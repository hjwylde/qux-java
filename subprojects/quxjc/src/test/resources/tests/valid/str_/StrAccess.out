b
a
r
q
b
